### ISSUE TITLE
Add support for Scala 2.11

### ISSUE THREAD

**Issue by ghost** [webcrank/webcrank-password.scala#12]
As the title says... needs to be updated to compile with Scala 2.11 so it can work with, for example, Play 2.3.

**Comment by markhibberd** [comment]
@emergentcypher Yep, sorry a bit slow with this one. I will push a release with this tomorrow.

**Comment by rpiaggio** [comment]
Hi, I'd really like to use this project in my Play 2.5 application, can you release it for Scala 2.11?

Thank you!

**Comment by danellis** [comment]
Issues are not being addressed -- 2.11 request is from 19 months ago. Is this project to be considered dead now?
